std_msgs/Header header
uint8 port
uint8 state
int8 power
int32 position
int16 dps